src/vmm/src/builder.rs

```diff
@@ -1352,7 +1339,12 @@ pub mod tests {
             let mut vmm = default_vmm();
             let mut cmdline = default_kernel_cmdline();
             insert_block_devices(&mut vmm, &mut cmdline, &mut event_manager, block_configs);
-            assert!(cmdline.as_str().contains("root=/dev/vda ro"));
+            assert!(cmdline
```

Comment:
This pattern repeats quite a lot. Maybe it would be worth adding a function for it

Reply:
I added a function that checks if cmdline contains a slug. Closing this comment for now. Thanks for the advice!